Dockerfile

```diff
@@ -36,4 +36,20 @@ FROM base as development
 ENTRYPOINT poetry run flask run --host=0.0.0.0 --port=8000
 
 ### Commands to build and run production image
+# See README file
+
+#--------------------------------------------------------------------------------#
+
+### Testing image...
+FROM base as test
+
+# Install pytest, flask and dotenv...
+RUN pip install pytest
+RUN pip install flask
+RUN pip install python-dotenv
```

Comment:
You should find these are unnecessary; you're already installing these libraries with poetry (when you run `poetry install`, since it checks what libraries are needed as specified in `pyproject.toml` and `poetry.lock`)